examples/asr/speech_to_text.py

```diff
@@ -62,7 +62,7 @@
 """
 
 
-@hydra.main(config_path="conf", config_name="config")
+@hydra_runner(config_path="conf", config_name="config")
```

Comment:
Do we need to expose the name "hydra" in the decorator? Why not "set_config" or sth similar?

My personal opinion is that we do not need to throw "hydra" in user's face in every main script...

Reply:
I want to explicitly show hydra here because this decorator is a facade over the hydra.main() decorator. The idea I want to tell the user is - You are still using hydra, just with another decorator.

Reply:
And I am totally against that.

Do you envision any other place in the user code where the name "hydra" will appear?

Reply:
Lets discuss this on slack. The PR needs to finalize implementation details rather than naming convention

Reply:
Created a poll:

https://nvidia.slack.com/archives/CHKBFJZV2/p1595011645375400

Reply:
Thanks